sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -46,7 +47,7 @@ def _populate_attributes(self, **kwargs):
             is_session=bool(self._session_id)
         )
 
-        self._name = kwargs.get("client_identifier", "SBReceiver-{}".format(uuid.uuid4()))
+        self._name = kwargs.get("client_identifier") or "SBReceiver-{}".format(uuid.uuid4())
```

Comment:
same here with kwargs

Reply:
sounds good, will expand this in the py 3 typing pr